gcc/rust/rust-lang.cc

```diff
@@ -434,6 +435,31 @@ rust_localize_identifier (const char *ident)
 #define LANG_HOOKS_GIMPLIFY_EXPR grs_langhook_gimplify_expr
 #define LANG_HOOKS_EH_PERSONALITY grs_langhook_eh_personality
 
+#if CHECKING_P
```

Comment:
can this or the tests be moved to a rust-base62-test.h?

Reply:
It should defintely be possible as the C frontend is doing it. I was in a rush when finishing this yesterday. I was planning on giving it a look in a next PR where I'd add actual assertions for the base62 functions, but I can also do it here if you prefer

Reply:
The main `LANG_HOOKS_RUN_LANG_SELFTESTS` should indeed be defined here, including the `selftest::rust_tests` function (or `selftest::run_rust_tests`, to keep it similar to C, C++?).  The individual tests (like for base62) should then reside in the respective files.  The `simple_assert` test here should eventually go away; but OK to leave it for now, if you'd like.